styles/mixins/input.css

```diff
@@ -27,4 +27,7 @@ title: Input
     border-bottom: 1px dashed $borderDisabled;
     color: $colorDisabled;
   }
+  &:read-only {
+    cursor: text;
```

Comment:
I thought about that but it didn't make much sense for cases where we output a url for a user to copy. I did try using the disabled colors but that didn't really look right.